packages/app-store/stripepayment/api/callback.ts

```diff
@@ -39,5 +39,5 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     },
   });
 
-  res.redirect("/apps/installed");
+  res.redirect(getInstalledAppPath({ variant: "payment", slug: "stripe" }));
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.